xml/System.Collections.Frozen/FrozenSet`1.xml

```diff
@@ -840,10 +855,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="equalValue">To be added.</param>
-        <param name="actualValue">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="equalValue">The value to search for.</param>
```

Comment:
@gewarren looks like there might be a bug in mdoc which is preventing some TypeParameter items to show up. Is this a known issue?

Reply:
@carlossanlop This method has no type parameters as far as I can tell.